CONTRIBUTING-ACL2020.md

```diff
@@ -51,124 +51,193 @@ sudo pip3 install virtualenvwrapper
-
-
-
-## How to Create Pull Request
-
-
-
-### Create Pull Request in the Main Repository
-
-
+## How to Create a Pull Request for the Website
 
 As mentioned in the [GitHub help documentation](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request), creating a pull request can propose and collaborate on changes to a repository. Those changes will be proposed to a *branch*, which ensures that the ```master``` branch only contains finished and approved works. 
 
 
-
+### Change Code on a Branch 
+#### Create a Branch
 ```shell
 # pull the newest update from the master branch
 git pull
 # create local branch
 git checkout -b your_branch_name
 ```
 
-By now you have created a new local branch. Then you can make contributions to the content. Once you've finished, use the following command in the terminal to commit to the local branch.
-
+#### Save your Changes
+By now you have created a new local branch. Then you can make contributions to the content. 
+You can save your contributions on your local git by commiting.
 ```shell
 git add . # add all files
 git add file_changed # only add file the named file_changed
-git commit -am 'your update logs'
+git commit -am 'what i did in this commit'
+```
+
+#### Run Code Checks
+When you think you are good, you should check that you code passes the github checks
+```shell
+# Auto formats your code
+make format
+# Run all checks locally
+make
+```
+If the second command fails, you need to fix the errors it indicates (and commit your fixes). (A warning is not a problem, however an error must be fixed)
+
+#### Push your modifications
+Once you've finished, use the following command in the terminal to push your changes to the global repository, on your branch.
+
+```shell
 git push --set-upstream origin your_branch_name # setup the upstream
 ```
 
+### Opening a Pull Request 
+#### In the Main Repository
+
 Now you can open a pull request on the repository's website. Additionally, on the repository's page, you should click ```New pull request``` button near the ```Branch: master```. Then it will redirect to a new page. On this page, you need to be very carefully for the choices of base repository and head repository. Following the example shown in this document, the head repository should be ```acl-org/acl-2020-virtual-conference```, and ```compare: your_branch_name``` next to the head repository menu. The base repository should be ```acl-org/acl-2020-virtual-conference```.  Then you can click the green button ```Create pull request```. By clicking this button, you have created a new pull request and a collaborator will be notified and review your changes. The collaborator will discuss any further changes for your contributions. Once approved, your branch will be merged into ```master``` branch. Before you make any further contributions, don't forget to ```git pull``` to update your local files.
 
-### Create a Pull Request with a Fork
+#### With a Fork
```

Comment:
can you remove this paragraph maybe?
I think there is some issue with forking from our repo because of the private submodule (see https://github.com/acl-org/acl-2020-virtual-conference/pull/308#issuecomment-652209593)